plugins/polio/migrations/0243_auto_20251014_1340.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.25 on 2025-10-14 13:40
+
+from django.db import migrations
+
+
+DOSES_PER_VIAL = {
+    "mOPV2": 20,
+    "nOPV2": 50,
+    "bOPV": 20,
+}
+
+
+def add_doses_per_vial(apps, schema_editor):
+    OutgoingStockMovement = apps.get_model("polio", "OutgoingStockMovement")
+    IncidentReport = apps.get_model("polio", "IncidentReport")
+    DestructionReport = apps.get_model("polio", "DestructionReport")
+    EarmarkedStock = apps.get_model("polio", "EarmarkedStock")
+
+    outgoing_stock_movements = OutgoingStockMovement.objects.filter(doses_per_vial__isnull=True).prefetch_related(
+        "vaccine_stock"
+    )
+    incident_reports = IncidentReport.objects.filter(doses_per_vial__isnull=True).prefetch_related("vaccine_stock")
+    destruction_reports = DestructionReport.objects.filter(doses_per_vial__isnull=True).prefetch_related(
+        "vaccine_stock"
+    )
+    earmarked_stocks = EarmarkedStock.objects.filter(doses_per_vial__isnull=True).prefetch_related("vaccine_stock")
+
+    for movement in outgoing_stock_movements:
+        doses_per_vial = DOSES_PER_VIAL.get(movement.vaccine_stock.vaccine, 20)
+        movement.doses_per_vial = doses_per_vial
+        movement.save()
+
+    for incident_report in incident_reports:
+        doses_per_vial = DOSES_PER_VIAL.get(incident_report.vaccine_stock.vaccine, 20)
+        incident_report.doses_per_vial = doses_per_vial
+        incident_report.save()
+
+    for destruction_report in destruction_reports:
+        doses_per_vial = DOSES_PER_VIAL.get(destruction_report.vaccine_stock.vaccine, 20)
+        destruction_report.doses_per_vial = doses_per_vial
+        destruction_report.save()
+
+    for stock in earmarked_stocks:
+        doses_per_vial = DOSES_PER_VIAL.get(stock.vaccine_stock.vaccine, 20)
+        stock.doses_per_vial = doses_per_vial
+        stock.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("polio", "0242_destructionreport_doses_per_vial_and_more"),
+    ]
+
+    operations = [migrations.RunPython(add_doses_per_vial, migrations.RunPython.noop)]
```

Comment:
We can tell Django to get rid of this during the next squash

```suggestion
    operations = [migrations.RunPython(add_doses_per_vial, migrations.RunPython.noop, elidable=True)]
```